src/QuantumOpticsBase.jl

```diff
@@ -3,31 +3,46 @@ module QuantumOpticsBase
 using SparseArrays, LinearAlgebra, LRUCache, Strided, UnsafeArrays
 import LinearAlgebra: mul!, rmul!
 
-export bases, Basis, GenericBasis, CompositeBasis, basis,
+export Basis, GenericBasis, CompositeBasis, basis,
         tensor, ⊗, permutesystems, @samebases,
-        states, StateVector, Bra, Ket, basisstate, sparsebasisstate, norm,
+        #states
+                StateVector, Bra, Ket, basisstate, sparsebasisstate, norm,
                 dagger, normalize, normalize!,
-        operators, AbstractOperator, DataOperator, expect, variance,
-            identityoperator, ptrace, reduced, embed, dense, tr, sparse,
-        operators_dense, Operator, DenseOperator, DenseOpType, projector, dm,
-        operators_sparse, SparseOperator, diagonaloperator, SparseOpType,
-        operators_lazysum, LazySum,
-        operators_lazyproduct, LazyProduct,
-        operators_lazytensor, LazyTensor, lazytensor_use_cache, lazytensor_clear_cache,
-        lazytensor_cachesize, lazytensor_disable_cache, lazytensor_enable_cache,
-        superoperators, SuperOperator, DenseSuperOperator, DenseSuperOpType,
+        #operators
```

Comment:
`operators` and other commented out symbols did not actually exist... I assume they were used to comment the code, saying where something is coming from.